src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,18 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static final int ARRAY_SIZE = 6;
 
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[ARRAY_SIZE];
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. It's good that you have created the `FigureSupplier` instance outside of the loop.